src/content/docs/infrastructure/prometheus-integrations/integrations-list/redpanda-integration.mdx

```diff
@@ -0,0 +1,95 @@
+* Install our [Prometheus OpenMetrics integration](/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-update-or-uninstall-your-prometheus-openmetrics-integration).
+
+## Configure Prometheus OpenMetrics for RedPanda [#configure]
+
+After you've installed Prometheus OpenMetrics, you need to configure the `nri-prometheus-config.yml` file to scrape for RedPanda data. Your configuration file should match our snippet in the [nri-prometheus](https://github.com/newrelic/nri-prometheus/blob/main/configs/nri-prometheus-config.yml.sample) repository: 
+
+```yml
+integrations:
+- name: nri-prometheus
+    config:
+      # When standalone is set to false, the infrastructure agent will need to send data so nri-prometheus can work. Defaults to true.
+      standalone: false
+
+      # If using the infrastructure agent, emitters must be set to infra-sdk
+      emitters: infra-sdk
+
+      # The name of your cluster. The name of your cluster must be consistent across New Relic products so the infrastructure agent and Prometheus OpenMetrics can scrape data from the cluster.
+      cluster_name: "YOUR_EXPORTER_NAME"
+
+      targets:
+        - description: "Redpanda metrics are captured in the below urls"
+          urls: ["'localhost:9644/metrics', 'localhost:9644/public_metrics']
+      # tls_config:
+      # ca_file_path: "/etc/etcd/etcd-client-ca.crt"
+      # cert_file_path: "/etc/etcd/etcd-client.crt"
+      # key_file_path: "/etc/etcd/etcd-client.key"
+
+      # Specifies whether or not the integration should run in verbose mode. Defaults to false.
+      verbose: false
+
+      # Specifies whether or not the integration should run in audit mode. Defaults to false.
+      # Audit mode logs the uncompressed data sent to New Relic. Use this to log all data sent.
+      # It does not include verbose mode. This can lead to a high log volume, so use with care.
+      audit: false
+
+      # The HTTP client timeout when fetching data from endpoints. Defaults to 30s.
+      # scrape_timeout: "30s"
+
+      # Length in time to distribute the scraping from the endpoints.
+      scrape_duration: "5s"
+
+      # Number of worker threads used for scraping targets.
+      # For large clusters with more than 400 endpoints, we recommend that you slowly increase until scrape time falls between the desired `scrape_duration`.
+      # Increasing this value too much will result in huge memory consumption if too many metrics are being scraped.
+      # Default: 4
+      # worker_threads: 4
+
+      # Specifies whether or not the integration should skip TLS verification. Defaults to false.
+      insecure_skip_verify: false
+
+    timeout: 10s
+```
+
+## Get RedPanda metrics as a dashboard
+
+Once you've installed the [Redpanda quickstart](https://newrelic.com/instant-observability/redpanda), you can see your critical Redpanda data in New Relic. To confirm that your data is reporting:
```

Comment:
The Redpanda quickstart link doesn't work. I imagine that it's because hasn't been released yet, but just in case, I let you know...